app/services/serialize_offer.rb

```diff
@@ -0,0 +1,49 @@
+class SerializeOffer
```

Comment:
Not a good name, but this is the service that takes structured condition params (including ids) that look like:
```
{
  '0' => { 'text' => 'DBS', 'condition_id' => '123' },
  '1' => { 'text' => 'Be cool', 'condition_id' => nil },
}
```

and updates existing conditions in the db

Reply:
There will need to be a version of this service that takes a list of strings and saves conditions too, so we can avoid making API changes. It's likely to look the same as UpdateOfferConditions